scss/nhse.scss

```diff
@@ -623,6 +623,19 @@ body {
   }
 }
 
+/**
+ * Main header navigation menu
+ */
+a.nhsuk-header__navigation-link.dropdown-item {
+    &:visited {
+        color: $gray-900;
```

Comment:
Font colour should be NHS blue and the highlight is just underline ... as per https://nhsuk.github.io/nhsuk-frontend/components/header/header-navigation.html 

![image](https://github.com/user-attachments/assets/997fcf3b-2386-4bc2-9111-3003b940e9fc)